src/DDTrace/Bootstrap.php

```diff
@@ -88,10 +88,10 @@ private static function initRootSpan()
         $startSpanOptions = 'cli' === PHP_SAPI
             ? StartSpanOptions::create($options)
             : StartSpanOptionsFactory::createForWebRequest(
-                $tracer,
-                $options,
-                Request::getHeaders()
-            );
+            $tracer,
+            $options,
+            Request::getHeaders()
+        );
```

Comment:
This doesn't look right -- have you investigated whether this is really what the standard would want? I suspect another bug in PHPCS.

Reply:
You were right! Looks like it got fixed in squizlabs/PHP_CodeSniffer#2654